roles/confluent.kafka_broker/templates/server.properties.j2

```diff
@@ -69,11 +69,11 @@ zookeeper.set.acl=true
 confluent.schema.registry.url={% for host in groups['schema_registry'] %}{% if loop.index > 1%},{% endif %}{{ schema_registry_http_protocol }}://{{ host }}:{{ schema_registry_listener_port }}{% endfor %}
 
 {% if schema_registry_ssl_enabled|bool %}
-confluent.schema.registry.ssl.truststore.location={{kafka_broker_truststore_path}}
-confluent.schema.registry.ssl.truststore.password={{kafka_broker_truststore_storepass}}
-confluent.schema.registry.ssl.keystore.location={{kafka_broker_keystore_path}}
-confluent.schema.registry.ssl.keystore.password={{kafka_broker_keystore_storepass}}
-confluent.schema.registry.ssl.key.password={{kafka_broker_keystore_keypass}}
+confluent.ssl.truststore.location={{kafka_broker_truststore_path}}
```

Comment:
yikes so as it turns out these keystore/truststore props need to be added to connect to SR but also to get the molecule test to pass... its unclear under what conditions